components/bpmn/bpmn-explorer-web/src/web/js/actions.js

```diff
@@ -1005,3 +1004,446 @@ function generateReport(){
+
+                        for (var k = 0; k < taskList.data.length; k++) {
+
+                            var activityName  = taskList.data[k].name;
+                            var taskDefKey  = taskList.data[k].taskDefinitionKey;
+                            var activityType  = taskList.data[k].type;
+                            if (activityType == "userTask") {
+
+                                for (var j = 0; j < taskList2.data.length; j++) {
+
+                                    var activityId  = taskList2.data[j].activityId;
+                                    var startTime = taskList2.data[j].startTime;
+                                    var endTime = taskList2.data[j].endTime;
+                                    var assignee = taskList2.data[j].assignee;
+                                    var duration =  taskList2.data[j].durationInMillis;
+
+                                    if (taskDefKey == activityId && endTime !== null ) {
+                                        var state = "Completed";
+
+                                        break;
+                                    } else if (taskDefKey == activityId && endTime == null ) {
+                                        var state = "Active";
+                                        var endTime = "N/A";
+                                        var duration = "N/A";
+                                        if (assignee == null) {
+                                            assignee = "Unassigned";
+                                        }
+
+                                        break;
+                                    } else {
+                                        var state = "Not Started";
+                                        var startTime = "N/A";
+                                        var endTime = "N/A";
+                                        var duration = "N/A";
+                                        var assignee = "Unassigned";
+
+                                    }
+                                }                
+                               
+                                DIV = DIV + "<tr><td>"+state+"</td><td style='word-wrap: break-word'>"+activityName+"</td><td style='word-wrap: break-word'>"+taskDefKey+"</td><td>"+startTime+"</td><td>"+endTime+"</td><td>"+duration+"</td><td>"+assignee+"</td></tr>";
+                            } else {
+                            }
+                        }
+                        DIV = DIV+"</tbody></table></div>"
+                        $("#userTasks").html(DIV);
+
+                    }
+                });
+            }
+    });
+}
+
+//Gets the details of any called process instance in a running/active process instance
+function getCalledProcessInstancesOfRunning(id){
+
+
```

Comment:
As a practice don't keep 2 new lines. If you want to have a new line to separate code blocks, just use a single line